config/config.yaml

```diff
@@ -39,6 +39,8 @@ kraken:
             - scenarios/zone_outage.yaml
         -   application_outages:
             - scenarios/app_outage.yaml
+        -   pvc_scenarios:
```

Comment:
Can you also add this to the config_performance.yaml file

Reply:
done